cw-orch-daemon/src/core.rs

```diff
@@ -380,7 +380,7 @@ pub async fn upload_wasm<T: TxSender>(
     e.write_all(&file_contents)?;
     let wasm_byte_code = e.finish()?;
     let store_msg = cosmrs::cosmwasm::MsgStoreCode {
-        sender: sender.account_id(),
+        sender: sender.msg_sender().map_err(Into::into)?,
```

Comment:
I don't think we need to authz for uploads

Reply:
why not ?

Reply:
Hmm interesting

Reply:
I am actually using it currently for Stargaze uploads :)

Reply:
cool! then LGMT :)